cluster-scope/bundles/hostpath-provisioner/scc.yaml

```diff
@@ -0,0 +1,25 @@
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: hostpath-provisioner
+allowPrivilegedContainer: false
+requiredDropCapabilities:
+- KILL
+- MKNOD
+- SETUID
+- SETGID
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+fsGroup:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+allowHostDirVolumePlugin: true
+users:
+- system:serviceaccount:kubevirt-hostpath-provisioner:kubevirt-hostpath-provisioner-admin
+volumes:
+- hostPath
+- secret
+- projected
```

Comment:
I don't see this included in any kustomization file.

Reply:
Well, that's because that file shouldn't be part of this PR :). Thanks!